.github/workflows/comment-bot.yml

```diff
@@ -15,4 +15,4 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             Thank you for creating a pull request!
-            In order to run the pipeline tests I require an admin to post the following comment: `run-tests`
+            In order to run the pipeline tests I require an admin to post the following comment: `run tests`
```

Comment:
When I did my PR "run-tests" worked but "run tests" did not have we changed it now to be consistent across repos?

Reply:
Just tried it and "run tests" I think did work

Reply:
The regex on the job hasn't changed `.*(re)?run tests.*`. Maybe you had commented `run tests` earlier in the PR and it picked up that